### DISCUSSION TITLE
How to Link to a Specific Part of video in MediaCMS?

### DISCUSSION THREAD

**Discussion by czhiling** [mediacms-io/mediacms#319] [Q&A] [answered]
Is there a way to link a specific part (timestamp) of the video in MediaCMS?

Thanks!

**Comment by mgogoulos** [comment, marked as answer]
Hello, no there isn't such a feature yet

**Reply by czhiling** [reply]
Thank you very much for the reply. I think it would be nice to have this function for references or comments by users.

**Reply by lil0r** [reply]
has this been added by now? the documentation https://github.com/mediacms-io/mediacms/blob/main/docs/user_docs.md#using-timestamps-for-sharing explicitly mentions timestamps. Doesn't seem to work at first glance.

**Reply by mgogoulos** [reply]
mmm can you open an issue for this so it gets some attention? It seems that it's not working indeed (althought it has been at some point)
